tests/contrib/test_minipyro.py

```diff
@@ -100,7 +115,7 @@ def guide(data):
     with pyro_backend(backend):
         Elbo = infer.JitTrace_ELBO if jit else infer.Trace_ELBO
         elbo = Elbo(ignore_jit_warnings=True)
-        assert_ok(model, guide, elbo, data)
+        assert_ok(model, guide, elbo, 2, data)
```

Comment:
nit: would you mind writing this in a self-documenting style
```py
steps = 2
assert_ok(model, guide, elbo, steps, data)
```
or maybe
```py
assert_ok(model, guide, steps=2, data=data)
```
or similar?